examples/snippets/project/_common/export_artifacts.py

```diff
@@ -0,0 +1,33 @@
+"""Snippet for exporting project artifacts from a Tamr project"""
+import tamr_toolbox as tbox
+from tamr_toolbox.project import categorization
+from tamr_toolbox.models.project_artifacts import CategorizationArtifacts as catfacts
```

Comment:
```suggestion
from tamr_toolbox.models.project_artifacts import CategorizationArtifacts
```

Prefer clarity over brevity

Reply:
preference acknowledged (and change accepted)